<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="authorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Author label</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="formComboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="formLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>315</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ukrword form</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nextPushButton">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>342</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="langLabel">
       <property name="text">
        <string>Language label</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="langComboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="aphorismLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Aphorism label</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="aphorismPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="authorLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <zorder>langComboBox</zorder>
   <zorder>aphorismLabel</zorder>
   <zorder>aphorismPlainTextEdit</zorder>
   <zorder>langLabel</zorder>
   <zorder>langLabel</zorder>
   <zorder>langLabel</zorder>
   <zorder>langComboBox</zorder>
   <zorder>authorLabel</zorder>
   <zorder>authorLineEdit</zorder>
   <zorder>aphorismLabel</zorder>
   <zorder>authorLineEdit</zorder>
   <zorder>aphorismPlainTextEdit</zorder>
   <zorder>formComboBox</zorder>
   <zorder>formLabel</zorder>
   <zorder>nextPushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
